diff --git a/configure b/configure
index 79532c7..39c3203 100755
--- a/configure
+++ b/configure
@@ -15907,7 +15907,7 @@ fi
 
 
 if test "$OPENLSWS_BSSL" = no ; then
-    if test "$OSNAME" = Darwin ; then
+    if test "$OSNAME" = Linux ; then
         usedynossl=yes
         CPPFLAGS="$CPPFLAGS -I../../src -I../../include -I/usr/local/include -I/usr/local -I/usr -I/usr/pkg "
     else
@@ -16392,7 +16392,7 @@ fi
 
 
 
-if test "$OSNAME" = Darwin ; then
+if test "$OSNAME" = Linux ; then
 
 
 # Check whether --with-openssl was given.
@@ -20970,5 +20970,3 @@ if test -n "$ac_unrecognized_opts" && test "$enable_option_checking" != no; then
   { $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: unrecognized options: $ac_unrecognized_opts" >&5
 $as_echo "$as_me: WARNING: unrecognized options: $ac_unrecognized_opts" >&2;}
 fi
-
-
