diff --git a/src/extensions/fcgi/fcgiconnection.cpp b/src/extensions/fcgi/fcgiconnection.cpp
index 184721d..be93b09 100644
--- a/src/extensions/fcgi/fcgiconnection.cpp
+++ b/src/extensions/fcgi/fcgiconnection.cpp
@@ -33,7 +33,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <unistd.h>
 
 const char FcgiConnection::s_padding[8] = { 0, 0, 0, 0, 0, 0, 0, 0 };
@@ -897,4 +897,3 @@ void FcgiConnection::dump()
             (m_lReqSentTime) ? time(NULL) - m_lReqSentTime : 0);
 
 }
-
