diff --git a/dist/functions.sh b/dist/functions.sh
index b4c4a38..4967478 100755
--- a/dist/functions.sh
+++ b/dist/functions.sh
@@ -45,7 +45,7 @@ init()
 
     DIR_OWN=$WS_USER:$WS_GROUP
     CONF_OWN=$WS_USER:$WS_GROUP
-    
+
 }
 
 license()
@@ -53,21 +53,21 @@ license()
 	more ./LICENSE
 	cat <<EOF
 
-IMPORTANT: In order to continue installation you must agree with above 
-           license terms by typing "Yes" with capital "Y"! 
+IMPORTANT: In order to continue installation you must agree with above
+           license terms by typing "Yes" with capital "Y"!
 
 EOF
 	SUCC=0
 	TRY=1
 	while [ $SUCC -eq "0" ]; do
-		printf "%s" "Do you agree with above license? "  
+		printf "%s" "Do you agree with above license? "
 		read YES_NO
 		if [ "x$YES_NO" != "xYes" ]; then
 			if [ $TRY -lt 3 ]; then
 				echo "Sorry, wrong answer! Type 'Yes' with capital 'Y', try again!"
 				TRY=`expr $TRY + 1`
 			else
-				echo "Abort installation!" 
+				echo "Abort installation!"
 				exit 0
 			fi
 
@@ -115,11 +115,11 @@ install_dir()
             DEST_RECOM="/opt/lsws"
         fi
 		WS_USER="nobody"
-	else 
+	else
 		cat <<EOF
 
 As you are not the 'root' user, you may not be able to install the
-web server into a system directory or enable chroot, the web server 
+web server into a system directory or enable chroot, the web server
 process will run on behalf of current user - '$INST_USER'.
 
 EOF
@@ -132,8 +132,8 @@ EOF
 	while [ $SUCC -eq "0" ];  do
 		cat <<EOF
 
-Please specify the destination directory. You must have permissions to 
-create and manage the directory. It is recommended to install the web server 
+Please specify the destination directory. You must have permissions to
+create and manage the directory. It is recommended to install the web server
 at /opt/lsws, /usr/local/lsws or in your home directory like '~/lsws'.
 
 ATTENTION: The user '$WS_USER' must be able to access the destination
@@ -146,7 +146,7 @@ EOF
 		if [ "x$TMP_DEST" = "x" ]; then
 			TMP_DEST=$DEST_RECOM
 		fi
-		if [ `expr "$TMP_DEST" : '~'` -gt 0 ]; then 
+		if [ `expr "$TMP_DEST" : '~'` -gt 0 ]; then
 			LSWS_HOME="$HOME`echo $TMP_DEST | sed 's/^~//' `"
 		else
 			LSWS_HOME=$TMP_DEST
@@ -165,11 +165,11 @@ EOF
 		if [ -f "$LSWS_HOME/conf/httpd_config.conf" ]; then
 			cat <<EOF
 
-Found old configuration file under destination directory $LSWS_HOME. 
+Found old configuration file under destination directory $LSWS_HOME.
 
 To upgrade, press 'Enter', current configuration will not be changed.
 To reinstall, press 'R' or 'r'.
-To change directory, press 'C' or 'c'. 
+To change directory, press 'C' or 'c'.
 
 EOF
 
@@ -179,15 +179,15 @@ EOF
 			if [ "x$TMP_URC" = "x" ]; then
 				INSTALL_TYPE="upgrade"
 				SET_LOGIN=0
-			else 
+			else
 				if [ `expr "$TMP_URC" : '[Uu]'` -gt 0 ]; then
 					INSTALL_TYPE="upgrade"
 					SET_LOGIN=0
-				else 
+				else
 					if [ `expr "$TMP_URC" : '[Rr]'` -gt 0 ]; then
 						INSTALL_TYPE="reinstall"
 						SET_LOGIN=1
-					else 
+					else
 					#if [ `expr "$TMP_URC" : '[Cc]'` -gt 0 ]; then
 						SUCC=0
 					fi
@@ -225,7 +225,7 @@ admin_login()
 				SET_LOGIN=1
 			fi
 		fi
-	fi		
+	fi
 
 	if [ $SET_LOGIN -eq 1 ]; then
 
@@ -300,7 +300,7 @@ a non-system user who does not have login shell and home directory such as
 'nobody'.
 
 EOF
-# get user name 
+# get user name
 		SUCC=0
 		while [ $SUCC -eq "0" ]; do
 			printf "%s" "User [$WS_USER]: "
@@ -312,22 +312,22 @@ EOF
 			TST_USER=`expr "$USER_INFO" : 'uid=.*(\(.*\)) gid=.*'`
 			if [ "x$TST_USER" = "x$TMP_USER" ]; then
 				USER_ID=`expr "$USER_INFO" : 'uid=\(.*\)(.*) gid=.*'`
-				if [ $USER_ID -gt 10 ]; then  
+				if [ $USER_ID -gt 10 ]; then
 					WS_USER=$TMP_USER
 					SUCC=1
 				else
 					cat <<EOF
 
-[ERROR] It is not allowed to run LiteSpeed web server on behalf of a 
-privileged user, user id must be greater than 10. The user id of user 
+[ERROR] It is not allowed to run LiteSpeed web server on behalf of a
+privileged user, user id must be greater than 10. The user id of user
 '$TMP_USER' is '$USER_ID'.
 
 EOF
 				fi
 			else
-				cat <<EOF 
+				cat <<EOF
 
-[ERROR] '$TMP_USER' is not valid user name in your system, please choose 
+[ERROR] '$TMP_USER' is not valid user name in your system, please choose
 another user or create user '$TMP_USER' first.
 
 EOF
@@ -336,7 +336,7 @@ EOF
 		done
 	fi
 
-# get group name 
+# get group name
 	SUCC=0
 	TMP_GROUPS=`groups $WS_USER`
 	TST_GROUPS=`expr "$TMP_GROUPS" : '.*:\(.*\)'`
@@ -347,7 +347,7 @@ EOF
 	D_GROUP=`$ID_GROUPS $WS_USER`
 	D_GROUP=`expr "$D_GROUP" : '.*gid=[0-9]*(\(.*\)) groups=.*'`
 	echo "Please choose the group that the web server running as."
-	echo 
+	echo
 	while [ $SUCC -eq "0" ];  do
 		echo "User '$WS_USER' is the member of following group(s): $TST_GROUPS"
 		printf "%s" "Group [$D_GROUP]: "
@@ -363,7 +363,7 @@ EOF
 			cat <<EOF
 
 [ERROR] '$TMP_GROUP' is not valid group for user '$WS_USER', please choose
-another group in the list or add user '$WS_USER' to group '$TMP_GROUP' 
+another group in the list or add user '$WS_USER' to group '$TMP_GROUP'
 first.
 
 EOF
@@ -385,7 +385,7 @@ stopLshttpd()
 	RUNNING_PROCESS=`$PS_CMD | grep lshttpd | grep -v grep`
 	if [ "x$RUNNING_PROCESS" != "x" ]; then
 		cat <<EOF
-LiteSpeed web server is running, in order to continue installation, the server 
+LiteSpeed web server is running, in order to continue installation, the server
 must be stopped.
 
 EOF
@@ -415,7 +415,7 @@ getServerPort()
 	cat <<EOF
 
 Please specify the port for normal HTTP service.
-Port 80 is the standard HTTP port, only 'root' user is allowed to use 
+Port 80 is the standard HTTP port, only 'root' user is allowed to use
 port 80, if you have another web server running on port 80, you need to
 specify another port or stop the other web server before starting LiteSpeed
 Web Server.
@@ -513,9 +513,9 @@ configAdminEmail()
 
 Please specify administrators' email addresses.
 It is recommended to specify a real email address,
-Multiple email addresses can be set by a comma 
+Multiple email addresses can be set by a comma
 delimited list of email addresses. Whenever something
-abnormal happened, a notificiation will be sent to 
+abnormal happened, a notificiation will be sent to
 emails listed here.
 
 EOF
@@ -550,9 +550,9 @@ enablePHPHandler()
 	cat <<EOF
 
 You can setup a global script handler for PHP with the pre-built PHP engine
-shipped with this package now. The PHP engine runs as Fast CGI which  
-outperforms Apache's mod_php. 
-You can always replace the pre-built PHP engine with your customized PHP 
+shipped with this package now. The PHP engine runs as Fast CGI which
+outperforms Apache's mod_php.
+You can always replace the pre-built PHP engine with your customized PHP
 engine.
 
 EOF
@@ -568,7 +568,7 @@ EOF
 	fi
 	if [ $SETUP_PHP -eq 1 ]; then
 		PHP_SUFFIX="php"
-		printf "%s" "Suffix for PHP script(comma separated list) [$PHP_SUFFIX]: " 
+		printf "%s" "Suffix for PHP script(comma separated list) [$PHP_SUFFIX]: "
 		read TMP_SUFFIX
 		if [ "x$TMP_SUFFIX" != "x" ]; then
 			PHP_SUFFIX=$TMP_SUFFIX
@@ -594,7 +594,7 @@ buildConfigFiles()
 {
 
 #sed -e "s/%ADMIN_PORT%/$ADMIN_PORT/" -e "s/%PHP_FCGI_PORT%/$ADMIN_PHP_PORT/" "$LSINSTALL_DIR/admin/conf/admin_config.conf.in" > "$LSINSTALL_DIR/admin/conf/admin_config.conf"
-    
+
     if [ "x${SSL_HOSTNAME}" != "x" ] ; then
         echo "SSL host is [${SSL_HOSTNAME}], use adminSSL"
         sed -e "s/%ADMIN_PORT%/$ADMIN_PORT/" -e "s/%SSL_HOSTNAME%/$SSL_HOSTNAME/" "$LSINSTALL_DIR/admin/conf/admin_config_ssl.conf.in" > "$LSINSTALL_DIR/admin/conf/admin_config.conf"
@@ -602,7 +602,7 @@ buildConfigFiles()
         echo "SSL host is [${SSL_HOSTNAME}], No adminssl"
         sed -e "s/%ADMIN_PORT%/$ADMIN_PORT/" "$LSINSTALL_DIR/admin/conf/admin_config.conf.in" > "$LSINSTALL_DIR/admin/conf/admin_config.conf"
     fi
-    
+
 	sed -e "s/%USER%/$WS_USER/" -e "s/%GROUP%/$WS_GROUP/" -e "s/%ADMIN_EMAIL%/$ADMIN_EMAIL/" -e "s/%HTTP_PORT%/$HTTP_PORT/" -e  "s/%RUBY_BIN%/$RUBY_PATH/" -e "s/%SERVER_NAME%/$SERVER_NAME/" "$LSINSTALL_DIR/conf/httpd_config.conf.in" > "$LSINSTALL_DIR/conf/httpd_config.conf.tmp"
 
 	if [ $SETUP_PHP -eq 1 ]; then
@@ -629,7 +629,7 @@ util_mkdir()
 	  chown "$OWNER" "$LSWS_HOME/$arg"
 	  chmod $PERM  "$LSWS_HOME/$arg"
 	done
-	
+
 }
 
 
@@ -641,13 +641,13 @@ util_cpfile()
 	shift
 	for arg
 	  do
-	  if [ -f "$LSINSTALL_DIR/$arg" ]; then 
+	  if [ -f "$LSINSTALL_DIR/$arg" ]; then
 	    cp -f "$LSINSTALL_DIR/$arg" "$LSWS_HOME/$arg"
 	    chown "$OWNER" "$LSWS_HOME/$arg"
 	    chmod $PERM  "$LSWS_HOME/$arg"
 	  fi
 	done
-	
+
 }
 
 util_ccpfile()
@@ -807,7 +807,7 @@ util_cpfilev()
 	for arg
 	  do
 	  if [ -f "$LSINSTALL_DIR/$arg" ]; then
-	  
+
         cp -f "$LSINSTALL_DIR/$arg" "$LSWS_HOME/$arg.$VERSION"
         chown "$OWNER" "$LSWS_HOME/$arg.$VERSION"
         chmod $PERM  "$LSWS_HOME/$arg.$VERSION"
@@ -835,31 +835,31 @@ create_lsadm_freebsd()
 {
     pw group add lsadm
     lsadm_gid=`grep "^lsadm:" /etc/group | awk -F : '{ print $3; }'`
-    pw user add -g $lsadm_gid -d / -s /usr/sbin/nologin -n lsadm 
+    pw user add -g $lsadm_gid -d / -s /usr/sbin/nologin -n lsadm
     pw usermod lsadm -G $WS_GROUP
 }
 
 create_lsadm()
 {
-    groupadd lsadm 
+    #groupadd lsadm
     #1>/dev/null 2>&1
     lsadm_gid=`grep "^lsadm:" /etc/group | awk -F : '{ print $3; }'`
-    useradd -g $lsadm_gid -d / -r -s /sbin/nologin lsadm 
-    usermod -a -G $WS_GROUP lsadm
+    #useradd -g $lsadm_gid -d / -r -s /sbin/nologin lsadm
+    #usermod -a -G $WS_GROUP lsadm
     #1>/dev/null 2>&1
-    
+
 }
 
 create_lsadm_solaris()
 {
-    groupadd lsadm 
+    groupadd lsadm
     #1>/dev/null 2>&1
     lsadm_gid=`grep "^lsadm:" /etc/group | awk -F: '{ print $3; }'`
-    useradd -g $lsadm_gid -d / -s /bin/false lsadm 
+    useradd -g $lsadm_gid -d / -s /bin/false lsadm
     usermod -G $WS_GROUP lsadm
 
     #1>/dev/null 2>&1
-    
+
 }
 
 
@@ -874,7 +874,7 @@ installation()
             create_lsadm_freebsd
         elif [ "x$SYS_NAME" = "xSunOS" ]; then
             create_lsadm_solaris
-        fi 
+        fi
         grep "^lsadm:" /etc/passwd 1>/dev/null 2>&1
         if [ $? -eq 0 ]; then
             CONF_OWN="lsadm:lsadm"
@@ -890,25 +890,25 @@ installation()
         rm -rf "$LSWS_HOME/admin/html.$VERSION"
     fi
 
-    util_mkdir "$SDIR_OWN" $DIR_MOD admin bin docs fcgi-bin php lib logs modules backup cachedata gdata docs/css docs/img docs/ja-JP docs/zh-CN admin/logs add-ons share share/autoindex share/autoindex/icons admin/fcgi-bin admin/html.$VERSION admin/misc tmp autoupdate 
+    util_mkdir "$SDIR_OWN" $DIR_MOD admin bin docs fcgi-bin php lib logs modules backup cachedata gdata docs/css docs/img docs/ja-JP docs/zh-CN admin/logs add-ons share share/autoindex share/autoindex/icons admin/fcgi-bin admin/html.$VERSION admin/misc tmp autoupdate
     util_mkdir "$CONF_OWN" $SDIR_MOD conf conf/cert conf/templates conf/vhosts conf/vhosts/Example admin/conf admin/tmp phpbuild
     util_mkdir "$SDIR_OWN" $SDIR_MOD admin/cgid admin/cgid/secret
     util_mkdir "$DIR_OWN" $SDIR_MOD tmp/ocspcache
     chgrp  $WS_GROUP $LSWS_HOME/admin/tmp $LSWS_HOME/admin/cgid
-    chmod  g+x $LSWS_HOME/admin/tmp $LSWS_HOME/admin/cgid 
+    chmod  g+x $LSWS_HOME/admin/tmp $LSWS_HOME/admin/cgid
     chown  $CONF_OWN $LSWS_HOME/admin/tmp/sess_* 1>/dev/null 2>&1
     chown  $DIR_OWN $LSWS_HOME/cachedata
-    util_mkdir "$SDIR_OWN" $DIR_MOD Example 
+    util_mkdir "$SDIR_OWN" $DIR_MOD Example
 
     find "$LSWS_HOME/admin/tmp" -type s -atime +1 -delete 2>/dev/null
     if [ $? -ne 0 ]; then
         find "$LSWS_HOME/admin/tmp" -type s -atime +1 2>/dev/null | xargs rm -f
-    fi 
+    fi
 
     find "/tmp/lshttpd" -type s -atime +1 -delete 2>/dev/null
     if [ $? -ne 0 ]; then
         find "/tmp/lshttpd" -type s -atime +1 2>/dev/null | xargs rm -f
-    fi 
+    fi
 
     WHM_CGIDIR="/usr/local/cpanel/whostmgr/docroot/cgi"
     if [ -d "$WHM_CGIDIR" ] && [ $INST_USER = "root" ]  ; then
@@ -916,15 +916,15 @@ installation()
     fi
     util_cpdir "$SDIR_OWN" $DOC_MOD add-ons
     util_cpdir "$CONF_OWN" $DOC_MOD share/autoindex
-    
+
     util_ccpfile "$SDIR_OWN" $EXEC_MOD fcgi-bin/lsperld.fpl fcgi-bin/RackRunner.rb
     util_cpfile "$SDIR_OWN" $EXEC_MOD  fcgi-bin/RailsRunner.rb  fcgi-bin/RailsRunner.rb.2.3
 	util_cpfile "$SDIR_OWN" $EXEC_MOD admin/misc/rc-inst.sh admin/misc/admpass.sh admin/misc/rc-uninst.sh admin/misc/uninstall.sh admin/misc/lsws.rc admin/misc/lsws.rc.gentoo admin/misc/enable_phpa.sh admin/misc/mgr_ver.sh admin/misc/gzipStatic.sh admin/misc/fp_install.sh admin/misc/create_admin_keypair.sh admin/misc/awstats_install.sh admin/misc/update.sh admin/misc/cleancache.sh admin/misc/lsup.sh
     util_cpfile "$SDIR_OWN" $EXEC_MOD admin/misc/ap_lsws.sh.in admin/misc/build_ap_wrapper.sh admin/misc/cpanel_restart_httpd.in admin/misc/build_admin_php.sh admin/misc/convertxml.sh
 	util_cpfile "$SDIR_OWN" $DOC_MOD admin/misc/gdb-bt admin/misc/htpasswd.php admin/misc/php.ini admin/misc/genjCryptionKeyPair.php admin/misc/purge_cache_byurl.php
     util_cpfile "$SDIR_OWN" $DOC_MOD admin/misc/convertxml.php
-    
-    
+
+
 	if [ $SET_LOGIN -eq 1 ]; then
 		util_cpfile "$CONF_OWN" $CONF_MOD admin/conf/htpasswd
 	else
@@ -950,7 +950,7 @@ installation()
         if [ ! -f "$LSWS_HOME/conf/vhosts/Example/vhconf.conf" ]; then
 			util_cpdir "$CONF_OWN" $CONF_MOD conf/vhosts/Example
 		fi
-		
+
 		#test if contains Example/html Example/cgi-bin and copy when installation
         if [ ! -d "$LSWS_HOME/Example/html" ]; then
             util_mkdir "$SDIR_OWN" $DIR_MOD Example/html
@@ -960,14 +960,14 @@ installation()
             util_mkdir "$SDIR_OWN" $DIR_MOD Example/cgi-bin
             util_cpdir "$SDIR_OWN" $DOC_MOD Example/cgi-bin
         fi
-		
+
 	else
 		util_cpfile "$CONF_OWN" $CONF_MOD admin/conf/admin_config.conf
 		util_cpfile "$CONF_OWN" $CONF_MOD conf/templates/ccl.conf conf/templates/phpsuexec.conf conf/templates/rails.conf
 		util_cpfile "$CONF_OWN" $CONF_MOD admin/conf/php.ini admin/conf/${SSL_HOSTNAME}.key admin/conf/${SSL_HOSTNAME}.crt
 		util_cpfile "$CONF_OWN" $CONF_MOD conf/httpd_config.conf conf/mime.properties conf/httpd_config.conf
         util_cpdir "$CONF_OWN" $CONF_MOD conf/vhosts/Example
-		util_mkdir "$SDIR_OWN" $DIR_MOD Example/html Example/cgi-bin 
+		util_mkdir "$SDIR_OWN" $DIR_MOD Example/html Example/cgi-bin
 		util_cpdir "$SDIR_OWN" $DOC_MOD Example/html Example/cgi-bin
 	fi
 
@@ -978,7 +978,7 @@ installation()
 
     util_cpfile "$CONF_OWN" $DOC_MOD conf/${SSL_HOSTNAME}.crt
     util_cpfile "$CONF_OWN" $DOC_MOD conf/${SSL_HOSTNAME}.key
-    
+
 	util_mkdir "$DIR_OWN" $DIR_MOD Example/logs Example/fcgi-bin
 	util_cpdir "$SDIR_OWN" $DOC_MOD admin/html.$VERSION
 	rm -rf $LSWS_HOME/admin/html
@@ -987,14 +987,14 @@ installation()
 
 	util_cpfile "$SDIR_OWN" $EXEC_MOD bin/updateagent
     util_cpfile "$SDIR_OWN" $EXEC_MOD bin/wswatch.sh
-	util_cpfilev "$SDIR_OWN" $EXEC_MOD $VERSION bin/lswsctrl bin/lshttpd 
+	util_cpfilev "$SDIR_OWN" $EXEC_MOD $VERSION bin/lswsctrl bin/lshttpd
 
     #if [ -e "$LSINSTALL_DIR/bin/lshttpd.dbg" ]; then
     #    if [ -f "$LSINSTALL_DIR/bin/lshttpd.dbg.$VERSION" ]; then
     #        rm "$LSINSTALL_DIR/bin/lshttpd.dbg.$VERSION"
     #    fi
     #    util_cpfilev "$SDIR_OWN" $EXEC_MOD $VERSION bin/lshttpd.dbg
-    #    
+    #
     #    #enable debug build for beta release
     #    ln -sf ./lshttpd.dbg.$VERSION $LSWS_HOME/bin/lshttpd
     #fi
@@ -1003,7 +1003,7 @@ installation()
     ln -sf lshttpd $LSWS_HOME/bin/litespeed
 
 	util_cpfile "$SDIR_OWN" $DOC_MOD docs/* docs/css/* docs/img/* docs/ja-JP/* docs/zh-CN/*
-	util_cpfile "$SDIR_OWN" $DOC_MOD VERSION GPL.txt  
+	util_cpfile "$SDIR_OWN" $DOC_MOD VERSION GPL.txt
 
 	if [ -f $LSWS_HOME/autoupdate/download ]; then
 		rm $LSWS_HOME/autoupdate/download
@@ -1025,7 +1025,7 @@ PHP scripts by caching them in compiled state, the overhead of compiling
 PHP is avoided.
 
 Note: If an opcode cache has been installed already, you do not need to
-      change it. If you need to built PHP binary by yourself, you need to 
+      change it. If you need to built PHP binary by yourself, you need to
       built PHP opcode cache from source as well, unless the version of your
       PHP binary is same as that the pre-built PHP opcode cache built for.
 
@@ -1069,7 +1069,7 @@ gen_selfsigned_cert()
     if [ "${SSL_ORG}" = "" ] ; then
 	SSL_ORG=LiteSpeedCommunity
     fi
-    
+
     if [ "${SSL_ORGUNIT}" = "" ] ; then
 	SSL_ORGUNIT=Testing
     fi
@@ -1120,23 +1120,23 @@ Congratulations! The LiteSpeed Web Server has been successfully installed.
 Command line script - "$LSWS_HOME/bin/lswsctrl"
 can be used to start or stop the server.
 
-It is recommended to limit access to the web administration interface. 
+It is recommended to limit access to the web administration interface.
 Right now the interface can be accessed from anywhere where this
-machine can be reached over the network. 
+machine can be reached over the network.
 
 Three options are available:
-  1. If the interface needs to be accessed only from this machine, just 
-     change the listener for the interface to only listen on the loopback 
-     interface - localhost(127.0.0.1). 
-  2. If the interface needs to be accessible from limited IP addresses or sub 
+  1. If the interface needs to be accessed only from this machine, just
+     change the listener for the interface to only listen on the loopback
+     interface - localhost(127.0.0.1).
+  2. If the interface needs to be accessible from limited IP addresses or sub
      networks, then set up access control rules for the interface accordingly.
-  3. If the interface has to be accessible via internet, SSL (Secure Sockets 
+  3. If the interface has to be accessible via internet, SSL (Secure Sockets
      Layer) should be used. Please read respective HOW-TOs on SSL configuration.
 
-To change configurations of the interface, login and click 
+To change configurations of the interface, login and click
 "Interface Configuration" button on the main page.
 The administration interface is located at http://localhost:<ADMIN_PORT>/
-or http://<ip_or_Hostname_of_this_machine>:<ADMIN_PORT>/ 
+or http://<ip_or_Hostname_of_this_machine>:<ADMIN_PORT>/
 
 EOF
 
@@ -1144,7 +1144,7 @@ EOF
 		if [ $INSTALL_TYPE != "upgrade" ]; then
 			printf "%s\n%s" "Would you like to have LiteSpeed Web Server started automatically" "when the server restarts [Y/n]? "
 			read START_SERVER
-			echo 
+			echo
 
 			if [ "x$START_SERVER" = "x" ]; then
 				START_SERVER=y
@@ -1153,7 +1153,7 @@ EOF
 				$LSWS_HOME/admin/misc/rc-inst.sh
 			else
 				cat <<EOF
-If you want to start the web server automatically later, just run 
+If you want to start the web server automatically later, just run
 	"$LSWS_HOME//rc-inst.sh"
 to install the service control script.
 
@@ -1161,12 +1161,12 @@ EOF
 			fi
 		fi
         if [ "x$HOST_PANEL" != "x" ]; then
-            cat << EOF 
+            cat << EOF
 
 The default configuration file contain support for both PHP4 and PHP5,
-A prebuilt PHP4 binary comes with this package, however, we recommend 
+A prebuilt PHP4 binary comes with this package, however, we recommend
 you to build your own PHP4 and PHP5 binaries though our web console with
-the same configuration parameters as your current PHP installation. You 
+the same configuration parameters as your current PHP installation. You
 can check your current PHP configuration via a phpinfo() page.
 
 Press [ENTER] to continue
@@ -1175,7 +1175,7 @@ EOF
 
             read TMP_VAL
 
-            cat << EOF 
+            cat << EOF
 
 When you replace Apache with LiteSpeed, remember to stop Apache completely.
 On most Linux servers, you should do:
@@ -1205,7 +1205,7 @@ EOF
 		printf "%s" "Would you like to restart it right now [Y/n]? "
 	fi
 	read START_SERVER
-	echo 
+	echo
 
 	if [ "x$START_SERVER" = "x" ]; then
 		START_SERVER=y
@@ -1243,4 +1243,3 @@ EOF
 	fi
 
 }
-
