diff --git a/src/http/httplistener.cpp b/src/http/httplistener.cpp
index 0344cdd..70b576c 100644
--- a/src/http/httplistener.cpp
+++ b/src/http/httplistener.cpp
@@ -40,7 +40,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <stdio.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/time.h>
 #include <unistd.h>
 #include <lsr/ls_strtool.h>
@@ -568,4 +568,3 @@ int HttpListener::mapDomainList(HttpVHost *pVHost, const char *pDomains)
     return m_pMapVHost->mapDomainList(pVHost, pDomains);
 
 }
-
