diff --git a/src/lsr/ls_lock.c b/src/lsr/ls_lock.c
index f0b36f0..dd88d01 100644
--- a/src/lsr/ls_lock.c
+++ b/src/lsr/ls_lock.c
@@ -79,11 +79,7 @@ int ls_pthread_mutex_setup(pthread_mutex_t *p)
 #else  /* defined(USE_MUTEX_ADAPTIVE) */
     /* pthread_mutexattr_settype(&myAttr, PTHREAD_MUTEX_NORMAL); */
     pthread_mutexattr_settype(&myAttr,
-#if defined(linux) || defined(__linux) || defined(__linux__) || defined(__gnu_linux__)
-                              PTHREAD_MUTEX_ERRORCHECK_NP
-#else  /* defined(linux) */
                               PTHREAD_MUTEX_ERRORCHECK
-#endif  /* defined(linux) */
                              );
 #endif  /* defined(USE_MUTEX_ADAPTIVE) */
     /* pthread_mutexattr_settype(&myAttr, PTHREAD_MUTEX_RECURSIVE); */
@@ -123,4 +119,3 @@ int ls_pspinlock_setup(ls_pspinlock_t   *p)
     }
     return LS_FAIL;
 }
-
