diff --git a/src/http/ntwkiolink.cpp b/src/http/ntwkiolink.cpp
index 5ad25d7..3a1e676 100644
--- a/src/http/ntwkiolink.cpp
+++ b/src/http/ntwkiolink.cpp
@@ -38,7 +38,7 @@
 #include <util/datetime.h>
 #include <util/stringtool.h>
 
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/uio.h>
@@ -1842,4 +1842,3 @@ int NtwkIOLink::isFromLocalAddr() const
     const struct sockaddr *pClient =  getClientInfo()->getAddr();
     return (GSockAddr::compareAddr(pServer, pClient) == 0);
 }
-
