diff --git a/src/thread/pthreadmutex.cpp b/src/thread/pthreadmutex.cpp
index 639b410..140627d 100644
--- a/src/thread/pthreadmutex.cpp
+++ b/src/thread/pthreadmutex.cpp
@@ -16,19 +16,5 @@
 *    along with this program. If not, see http://www.gnu.org/licenses/.      *
 *****************************************************************************/
 #include "pthreadmutex.h"
-#if (defined(linux) || defined(__linux) || defined(__linux__) || defined(__gnu_linux__)) && !defined( NDEBUG )
-
-#ifndef PTHREAD_ERRORCHECK_MUTEX_INITIALIZER_NP
-# define PTHREAD_ERRORCHECK_MUTEX_INITIALIZER_NP \
-    {0, 0, 0, PTHREAD_MUTEX_ERRORCHECK_NP, __LOCK_INITIALIZER}
-#endif
-
-pthread_mutex_t PThreadMutex::s_proto =
-    PTHREAD_ERRORCHECK_MUTEX_INITIALIZER_NP;
-
-#else
 
 pthread_mutex_t PThreadMutex::s_proto = PTHREAD_MUTEX_INITIALIZER;
-
-#endif
-
