--- a/config.m4
+++ b/config.m4
@@ -17,14 +17,14 @@
       AC_MSG_RESULT([supported ($PHP_XDEBUG_FOUND_VERSION)])
     fi
   fi
-  
+
   AC_DEFINE(HAVE_XDEBUG,1,[ ])
 
   old_CPPFLAGS=$CPPFLAGS
   CPPFLAGS="$INCLUDES $CPPFLAGS"
 
   AC_CHECK_FUNCS(gettimeofday)
-  AC_CHECK_HEADERS([netinet/in.h])
+  AC_CHECK_HEADERS([netinet/in.h poll.h sys/poll.h])
 
   PHP_CHECK_LIBRARY(m, cos, [ PHP_ADD_LIBRARY(m,, XDEBUG_SHARED_LIBADD) ])
 
