diff --git a/dist/functions.sh b/dist/functions.sh
index b4c4a38..f721dd7 100755
--- a/dist/functions.sh
+++ b/dist/functions.sh
@@ -841,11 +841,11 @@ create_lsadm_freebsd()
 
 create_lsadm()
 {
-    groupadd lsadm 
+    #groupadd lsadm 
     #1>/dev/null 2>&1
     lsadm_gid=`grep "^lsadm:" /etc/group | awk -F : '{ print $3; }'`
-    useradd -g $lsadm_gid -d / -r -s /sbin/nologin lsadm 
-    usermod -a -G $WS_GROUP lsadm
+    #useradd -g $lsadm_gid -d / -r -s /sbin/nologin lsadm 
+    #usermod -a -G $WS_GROUP lsadm
     #1>/dev/null 2>&1
     
 }

