diff --git a/ext/snmp/snmp.c b/ext/snmp/snmp.c
index 7e299999a6..1605f0c6eb 100644
--- a/ext/snmp/snmp.c
+++ b/ext/snmp/snmp.c
@@ -41,9 +41,9 @@
 #if HAVE_SNMP
 
 #include <sys/types.h>
+#include <errno.h>
 #ifdef PHP_WIN32
 #include <winsock2.h>
-#include <errno.h>
 #include <process.h>
 #include "win32/time.h"
 #elif defined(NETWARE)
@@ -52,17 +52,11 @@
 #else
 #include <sys/socket.h>
 #endif
-#include <errno.h>
 #include <sys/timeval.h>
 #else
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
-#ifndef _OSD_POSIX
-#include <sys/errno.h>
-#else
-#include <errno.h>  /* BS2000/OSD uses <errno.h>, not <sys/errno.h> */
-#endif
 #include <netdb.h>
 #endif
 #ifdef HAVE_UNISTD_H
