--- a/ext/snmp/snmp.c
+++ b/ext/snmp/snmp.c
@@ -37,21 +37,16 @@
 
 #if HAVE_SNMP
 
+#include <errno.h>
 #include <sys/types.h>
 #ifdef PHP_WIN32
 #include <winsock2.h>
-#include <errno.h>
 #include <process.h>
 #include "win32/time.h"
 #else
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
-#ifndef _OSD_POSIX
-#include <sys/errno.h>
-#else
-#include <errno.h>  /* BS2000/OSD uses <errno.h>, not <sys/errno.h> */
-#endif
 #include <netdb.h>
 #endif
 #ifdef HAVE_UNISTD_H
