diff --git a/src/extensions/cgi/cgiconnection.cpp b/src/extensions/cgi/cgiconnection.cpp
index 864344c..1a97bbf 100644
--- a/src/extensions/cgi/cgiconnection.cpp
+++ b/src/extensions/cgi/cgiconnection.cpp
@@ -26,7 +26,7 @@
 #include <assert.h>
 #include <errno.h>
 #include <stdio.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/socket.h>
 #include <unistd.h>
 
@@ -202,4 +202,3 @@ void CgiConnection::operator delete(void *p)
     --s_iCgiCount;
     return Pool::deallocate(p, sizeof(CgiConnection));
 }
-
