diff --git a/src/extensions/cgi/lscgid.cpp b/src/extensions/cgi/lscgid.cpp
index 831f449..33532f5 100644
--- a/src/extensions/cgi/lscgid.cpp
+++ b/src/extensions/cgi/lscgid.cpp
@@ -34,7 +34,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/resource.h>
 #include <sys/stat.h>
 #include <sys/socket.h>
@@ -823,5 +823,3 @@ int lscgid_main(int fd, char *argv0, const char *secret, char *pSock)
     }
     return ret;
 }
-
-
