diff --git a/src/edio/pollfdreactor.cpp b/src/edio/pollfdreactor.cpp
index f0ee2ff..d8c4ca6 100644
--- a/src/edio/pollfdreactor.cpp
+++ b/src/edio/pollfdreactor.cpp
@@ -21,7 +21,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <assert.h>
 
 
@@ -129,6 +129,3 @@ int PollfdReactor::remove(EventReactor *pHandler)
     }
     return LS_FAIL;
 }
-
-
-
