diff --git a/configure b/configure
index 58d99a8..af01998 100755
--- a/configure
+++ b/configure
@@ -15874,7 +15874,7 @@ fi
 
 
 if test "$OPENLSWS_BSSL" = no ; then
-    if test "$OSNAME" = Darwin ; then
+    if test "$OSNAME" = Linux ; then
         usedynossl=yes
         CPPFLAGS="$CPPFLAGS -I../../src -I../../include -I/usr/local/include -I/usr/local -I/usr -I/usr/pkg "
     else
@@ -16142,7 +16142,7 @@ fi
 
 
 
-if test "$OSNAME" = Darwin ; then
+if test "$OSNAME" = Linux ; then
 
 
 # Check whether --with-openssl was given.
