diff --git a/dist/install.sh b/dist/install.sh
index 7b072da..0997d6f 100755
--- a/dist/install.sh
+++ b/dist/install.sh
@@ -43,12 +43,8 @@ inst_admin_php()
     fi
         
     if [ "x$OS" = "xLinux" ] ; then
-        if [ "x$OSTYPE" != "xx86_64" ] ; then
-            $DLCMD $LSWS_HOME/admin/fcgi-bin/admin_php http://www.litespeedtech.com/packages/lsphp5_bin/i386/lsphp5
-        else
-            $DLCMD $LSWS_HOME/admin/fcgi-bin/admin_php http://www.litespeedtech.com/packages/lsphp5_bin/x86_64/lsphp5
-        fi
-        
+        ln -s /usr/local/bin/lsphp $LSWS_HOME/admin/fcgi-bin/admin_php
+
         if [ $? = 0 ] ; then 
             HASADMINPHP=y
             echo "admin_php downloaded."
