diff --git a/src/thread/thread.h b/src/thread/thread.h
index 99bda16..2443562 100644
--- a/src/thread/thread.h
+++ b/src/thread/thread.h
@@ -87,11 +87,6 @@ public:
     int timedJoin(void **pRetVal, struct timespec *timeout)
     {   return pthread_timedjoin_np(m_tid, pRetVal, timeout);    }
 
-    int attrSetAffinity(size_t cpusetsize, const cpu_set_t *cpuset)
-    {   return pthread_attr_setaffinity_np(&m_attr, cpusetsize, cpuset);  }
-
-    int attrGetAffinity(size_t cpusetsize, cpu_set_t *pCpuSet)
-    {   return pthread_attr_getaffinity_np(&m_attr, cpusetsize, pCpuSet);  }
 #endif
 
     int attrSetDetachState(int detachstate)
