diff --git a/sapi/phpdbg/phpdbg.c b/sapi/phpdbg/phpdbg.c
index 121f1edb17..ccd1bf360f 100644
--- a/sapi/phpdbg/phpdbg.c
+++ b/sapi/phpdbg/phpdbg.c
@@ -39,7 +39,11 @@
 #	include <sys/select.h>
 #	include <sys/time.h>
 #	include <sys/types.h>
-#	include <sys/poll.h>
+#	if HAVE_POLL_H
+#		include <poll.h>
+#	elif HAVE_SYS_POLL_H
+#		include <sys/poll.h>
+#	endif
 #	include <netinet/in.h>
 #	include <unistd.h>
 #	include <arpa/inet.h>
@@ -1324,7 +1328,7 @@ void phpdbg_free_wrapper(void *p ZEND_FILE_LINE_DC ZEND_FILE_LINE_ORIG_DC) /* {{
 {
 	zend_mm_heap *heap = zend_mm_get_heap();
 	if (UNEXPECTED(heap == p)) {
-		/* TODO: heap maybe allocated by mmap(zend_mm_init) or malloc(USE_ZEND_ALLOC=0) 
+		/* TODO: heap maybe allocated by mmap(zend_mm_init) or malloc(USE_ZEND_ALLOC=0)
 		 * let's prevent it from segfault for now
 		 */
 	} else {
